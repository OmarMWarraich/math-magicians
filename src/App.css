.quote {
  font-size: 1.5rem;
  margin: 0;
  padding: 0;
  text-align: center;
}

.quote p {
  margin: 4rem;
  padding: 0;
  text-align: center;
}

.welcome {
  padding: 20px;
  border-radius: 5px;
  margin-bottom: 20px;
}

.welcome h1 {
  font-size: 3rem;
  font-weight: 700;
  margin-top: 5rem;
  padding: 0;
  text-align: center;
}

.welcome p {
  margin: 4rem;
  font-size: 2rem;
  padding: 0;
  text-align: center;
}

@media screen and (max-width: 768px) {

  .quote {
    font-size: 1rem;
    margin: auto;

  }

  .welcome {
    margin: auto;
  }

  .welcome h1 {
    font-size: 2rem;
  }

  .welcome p {
    margin: 2rem;
    font-size: 1rem;
  }

}
  
