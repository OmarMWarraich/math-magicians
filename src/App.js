import React from 'react';
import { Routes, Route } from 'react-router-dom';
import Calculator from './components/Calculator';
import Navbar from './components/Navbar';
import Quotes from './pages/Quotes';
import Welcome from './pages/Welcome';
import './App.css';

const App = () => (
  <>
    <Navbar />
    <Routes>
      <Route path="/" element={<Welcome />} />
      <Route path="/calculator" element={<Calculator />} />
      <Route path="/quote" element={<Quotes />} />
    </Routes>
  </>
);
export default App;
