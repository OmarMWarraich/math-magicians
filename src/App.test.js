import React from "react";
import { BrowserRouter } from "react-router-dom";
import { render } from "@testing-library/react";
import App from "./App";

describe('test App component', () => {
  test('renders App component', () => {
    const component = render(
    <BrowserRouter>
        <App />
    </BrowserRouter>
    );
    expect(component).toMatchSnapshot();
  });
});

