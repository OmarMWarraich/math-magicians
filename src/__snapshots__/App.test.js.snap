// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App component renders App component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="navbar"
      >
        <h1>
          Math Magicians
        </h1>
        <ul>
          <li>
            <a
              aria-current="page"
              class="active-link active"
              exact="false"
              href="/"
            >
              Welcome
            </a>
          </li>
          <li>
            <a
              class="active-link"
              exact="false"
              href="/calculator"
            >
              Calculator
            </a>
          </li>
          <li>
            <a
              class="active-link"
              exact="false"
              href="/quote"
            >
              Quote
            </a>
          </li>
        </ul>
      </div>
      <div
        class="welcome"
      >
        <h1>
          Welcome to our page!
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="navbar"
    >
      <h1>
        Math Magicians
      </h1>
      <ul>
        <li>
          <a
            aria-current="page"
            class="active-link active"
            exact="false"
            href="/"
          >
            Welcome
          </a>
        </li>
        <li>
          <a
            class="active-link"
            exact="false"
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            class="active-link"
            exact="false"
            href="/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </div>
    <div
      class="welcome"
    >
      <h1>
        Welcome to our page!
      </h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
